### PR TITLE
Access Control: Add an endpoint for setting several managed resource permissions

### PR DESCRIPTION
**Pull request by IevaVasiljeva**
**What is this feature?**

An endpoint for setting several managed resource permissions with one request.

**Why do we need this feature?**

It will be used by Terraform to provision RBAC setup, and users can also call it directly (through custom scripts etc).

### CODE REVIEW

**Review by kalleep — APPROVED** [review]
LGTM, lets wait for 9.2.3 cutoff before we merge this one :)

**Comment by IevaVasiljeva** [comment]
> LGTM, lets wait for 9.2.3 cutoff before we merge this one :)

Yeah, I'll merge it in the main, but hold off with the backport :) I've also updated the milestone to 9.2.4.

**Pull request merged**
